eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -68,8 +68,8 @@ jobs:
       - job:
         displayName: Create API Review for ${{ artifact.name }}
         pool:
-          name: azsdk-pool-mms-win-2019-general
-          vmImage: MMS2019
+          name: azsdk-pool-mms-win-2022-general
+          vmImage: MMS2022
```

Comment:
`vmImage` can be removed in the two places where it's updated to `MMS2022` here. It's not required.

Reply:
I'll clean this up in another issue. https://github.com/Azure/azure-sdk-for-cpp/issues/4211